nixos/modules/services/misc/matrix-synapse.nix

```diff
@@ -720,7 +720,10 @@ in {
           --keys-directory ${cfg.dataDir} \
           --generate-keys
       '';
-      environment.PYTHONPATH = makeSearchPathOutput "lib" cfg.package.python.sitePackages [ pluginsEnv ];
+      environment = {
+        LD_PRELOAD = "${pkgs.jemalloc}/lib/libjemalloc.so";
```

Comment:
This should have a flag to opt-in to this.

Reply:
It should be opt out because this is what upstream does in their Docker Image which is the most common method to install synapse. https://github.com/matrix-org/synapse/blob/develop/docker/start.py#L220